lib/src/cli.dart

```diff
@@ -35,12 +35,13 @@ cli(
     @Rest(help: getTaskHelp, allowed: allowedTasks)
     List<String> tasks,
     {
-     @Flag(help: 'Print the version of grinder.')
+     @Flag(help: 'Print the version of grinder.', defaultsTo: false)
      bool version,
```

Comment:
Nit: You can just do `bool version: false`.  I like to define as much of the interface using dart-native features as possible.